assets/banano_node_list.yml

```diff
@@ -0,0 +1,4 @@
+-
+  uri: kaliumapi.appditto.com/api
```

Comment:
this will be handled differently when cw-602 (#1356) is merged

Reply:
Yeah I added these and then realized that CW-602 needed to exist since they didn’t work

All the changes in CW-602 can safely overwrite anything conflicting here

so let’s prioritize CW-602 and once it’s merged I’ll discard these changes